CHANGELOG.md

```diff
@@ -13,6 +13,19 @@ during a frame which would not run a simulation step.
 its default value is `TriMeshFlags::MERGE_DUPLICATE_VERTICES`,
 which was its hardcoded behaviour.
 
+### Modified
+
+- `RapierContext`, `RapierConfiguration` and `RenderToSimulationTime` are now a `Component`
+  - Rapier now supports multiple worlds, see example `multi_world3` for usage details.
```

Comment:
```suggestion
- `RapierContext`, `RapierConfiguration` and `RenderToSimulationTime` are now a `Component` instead of resources.
  - Rapier now supports multiple independent physics worlds, see example `multi_world3` for usage details.
```